test/e2e/e2e-tests.sh

```diff
@@ -0,0 +1,43 @@
+TEST_NAMESPACE="openshift-marketplace"
+TEST_NAMESPACE_2="operator-test"
+MANIFEST_FOLDER="./test/e2e/environment/"
+NAMESPACED_MANIFEST="./${MANIFEST_FOLDER}/namespaced-manifest.yaml"
+GLOBAL_MANIFEST="./${MANIFEST_FOLDER}/global-manifest.yaml"
+
+OPERATOR_SOURCE_CRD="./deploy/crds/operatorsource.crd.yaml"
+CATALOG_SOURCE_CONFIG_CRD="./deploy/crds/catalogsourceconfig.crd.yaml"
+OPERATOR="./deploy/operator.yaml"
+SERVICE_ACCOUNT="./deploy/service_account.yaml"
+ROLE_BINDING="./deploy/role_binding.yaml"
+ROLE="./deploy/role.yaml"
+
+# Create openshift resources if they don't exist
+echo "Creating openshift resources"
+oc apply -f $OPERATOR_SOURCE_CRD
+oc apply -f $CATALOG_SOURCE_CONFIG_CRD
```

Comment:
Any reason not to use the `oc get --ignore-not-found` for the CRDs?

Reply:
oc apply does an update if the resource type already exists. the oc get for the namespaces is there because if the namespace already exists it will fail